app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
+
+  &:after {
+    font-size: em(10, 19);
+    position: absolute;
+    right: -15px;
```

Comment:
Minor: If there are any "magic numbers" like here (or below on the `margin-top`), I would try to make them relate to some other size (e.g. a fraction of the size of the arrow) and then explain that in a comment if it isn't obvious.